lib/zendesk_api/silent_mash.rb

```diff
@@ -0,0 +1,8 @@
+require 'hashie'
+
+module ZendeskAPI
+  # @private
+  class SilentMash < Hashie::Mash
+    disable_warnings
```

Comment:
Yeah I found out that I needed to bump hashie but it came as a bit of a surprise!